platformio.ini

```diff
@@ -9,71 +9,40 @@
 ; https://docs.platformio.org/page/projectconf.html
 
 [platformio]
-; this sets the base dir for src_filter in the board-specific envs
 src_dir = .
 
-
-;;
-;; Shared environments section
-;; Board-specific environments are below this section
-;;
-
 [base1549]
-; common configuration for all LPC1549 targets
 platform = nxplpc
 platform_packages = 
-  tool-openocd
+	tool-openocd
```

Comment:
Can you undo changes to this file? It looks like a lot of formatting changes (plus removal of comments). Especially if the library being pulled in doesn't actually work.

I suspect the formatting changes is PlatformIO being stupid and lazy when you try to alter the build configs through tools as opposed to manually editing the file - instead of inserting or modifying lines (which understandably is hard - been there, done that - but not THAT hard), they just re-serialize and re-write the entire file (and incidentally reformat everything, and discard all the useful comments).